adrs/0001-pseudo-private-custom-properties.md

```diff
@@ -0,0 +1,28 @@
+# Refactor component styling using pseudo-private custom properties
+
+**Date:** [2022-11-03]
```

Comment:
Maybe before merging the PR we should add a link here to this PR for future reference (see ADR template). 🙂

Reply:
```suggestion
**Date:** 2022-11-03

**Related PR:** #1172
```